jsonschema/_utils.py

```diff
@@ -55,6 +58,19 @@ def load_schema(name):
     return json.loads(data.decode("utf-8"))
 
 
+def load_vocabulary(name):
+    """
+    Load all schema files from ./schemas/``name`` and return them as a list.
+    """
+    vocabulary = []
+    base_path = os.path.dirname(sys.modules["jsonschema"].__file__)
```

Comment:
This function is suposed to cover the loading of all the references provided in the meta schema (otherwise the URLs will be fetched from remote on every test, which cases them to run about 30 min/pipeline). Would it be an option to be explicit about which vocabularies should be loaded? This way the paths can be provided when creating the validator.